rosdep/python.yaml

```diff
@@ -5350,6 +5350,10 @@ python3-can:
   ubuntu:
     '*': [python3-can]
     xenial: null
+python3-catkin-lint:
+  debian: [python3-catkin-lint]
```

Comment:
We're going to have to call this out separately for bullseye, since it isn't supported in the other currently-supported Debian distributions.  Something like this should do:

```
debian:
  '*': [python3-catkin-lint]
  jessie: null
  stretch: null
  buster: null
```